jax/_src/numpy/ufuncs.py

```diff
@@ -1744,11 +1744,39 @@ def _wrap_between(x, _a):
   return lax.sub(rem, a)
 
 
-@custom_jvp
-@implements(np.logaddexp2, module='numpy')
 @jit
 def logaddexp2(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Logarithm of the sum of exponentiations of the inputs in base-2.
+
+  JAX implementation of :obj:`numpy.logaddexp2`.
+
```

Comment:
We should mention somewhere that this is implemented carefully to work correctly in cases where the naive version would overflow.

Reply:
Changed the API description to `"Logarithm of the sum of exponentials of inputs in base-2 avoiding overflow."`

Reply:
Looks good, thanks!